### DISCUSSION TITLE
Chirp cheepcheep cheep chirp tweet DreamBerd chirp

### DISCUSSION THREAD

**Discussion by Magnogen** [TodePond/GulfOfMexico#34] [Q&A] [answered]
Cheep cheep chirp, tweetwoo chirp chip chirp chirrrp.

Chirp chirip chirp cheepchirp cheep cheep? Cheep chirp cheep! Cheepcheep tweet tweetwoo:

```
cheepcheep main(a) => {
   chirp 'Hello, ' + a!
}
main('World!')?
```

Cheep chirp tweet, chirp tweet tweet chirp.

Tweet tweet chip?

**Comment by Magnogen** [comment]
Cheep cheep chirp, tweet chip chip. Chirp!

```
chip cheep i: Int!

tweet (i % 3 = 0 && i % 5 = 0) "CheepChirp"?
tweetwoo (i % 3 = 0) "Cheep"?
tweetwoo (i % 5 = 0) "Chirp"?
twoo i? 

tweet (i < 20) i++!
i = 0!
```

**Comment by TodePond** [comment, marked as answer]
Cheep

**Comment by Magnogen** [comment]
Cheep cheep!

**Comment by D-u-c-k-s-e-l** [comment]
(in bird) I think this breaks the Twitter API.
